scripts/generate-json-schema.ts

```
@@ -210,42 +220,65 @@
+			}),
+			Result.inspectError((error) => {
+				logger.error(`Failed to write docs/public/${SCHEMA_FILENAME}:`, error);
+				process.exit(1);
+			}),
+			Result.inspect(() => logger.info(`✓ Generated docs/public/${SCHEMA_FILENAME}`)),
+		);
+	}
+
+	// Run lint only on files that were actually changed
+	if (filesToLint.length > 0) {
+		await Result.pipe(
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)